redis/src/commands/json_macro.rs

```diff
@@ -0,0 +1,145 @@
+macro_rules! implement_json_commands {
```

Comment:
Let's move this into the `json` module, somewhere near the bottom.

Reply:
> Let's move this into the `json` module, somewhere near the bottom.

I can put it at the top, but it complains if I put it at the bottom since it's *technically* being declared after use

Reply:
Ah, fair. At the top it is!